.github/workflows/build.yml

```diff
@@ -123,38 +97,38 @@ jobs:
         name: logs-${{ matrix.vcpkg_triplet }}
         path: ${{ matrix.vcpkg_path  }}/buildtrees/**/*.log
 
-    - name: Create buildenv archive
-      run: ./vcpkg export --vcpkg-root=${{ matrix.vcpkg_path }} --x-all-installed --zip --output=${{ env.DEPS_BASE_NAME }}-${{ env.MIXXX_VERSION }}-${{ matrix.vcpkg_triplet }}-${{ steps.vars.outputs.sha_short }}
-      working-directory: ${{ matrix.vcpkg_path }}
+    #- name: Create buildenv archive
```

Comment:
For a first step of transition it might be good to generate the old style buildenv zip.
Is it correct, that we could keep the scripts in mixxx/tools unchanged, if we leave this section active?

Reply:
Yes, currently the zip is created manually below, because export did not work in Manifest Mode.

I think this is now supported. We need to test.